pkg/operator/operator.go

```diff
@@ -194,6 +195,17 @@ func Start(ctx context.Context, buildInfo trivyoperator.BuildInfo, operatorConfi
 			return fmt.Errorf("unable to setup clustercompliancereport reconciler: %w", err)
 		}
 	}
+
```

Comment:
I would make this configurable (can be default true), in case one wants to use an external metrics workload

Reply:
I agree, and have added configuration (code) for this now. Docs needs to added/updated, but let's agree on the code first. I might need some pointers to how I should add this new feature to the docs...